docs/reach/rs2/reachview/mobile-data.md

```diff
@@ -10,6 +10,8 @@ Enable Mobile data on Reach to [get corrections from NTRIP over the Internet](ht
 
 	</center>
 
+[Learn more about connection Reach RS2 to the Internet via a built-in cellular modem.](https://docs.emlid.com/reachrs2/connecting-to-the-internet/#video-guide)
```

Comment:
[Learn more about connecting Reach RS2 to the Internet in this docs entry.](https://docs.emlid.com/reachrs2/connecting-to-the-internet/)

Please, move it below the note